src/core/Atom.ts

```diff
@@ -42,6 +42,11 @@ export default class Atom {
     );
   }
 
+  remapSiteInplace(map: IndexMap): void {
```

Comment:
This function is unused, according to coverage. Please add a note or TODO that it is still not being used, so that we don't forget about it.